lib/collection/request-body.js

```diff
@@ -61,6 +61,16 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
 
         _.isString(options.file) && (file = { src: file });
 
+        if (!raw) {
```

Comment:
why is this needed? + line 65 does close to this, followed subsequent removal of raw in the _.assign below

Reply:
This handles raw bodies regardless of the current request body mode. The condition below is to sanitize raw bodies only if a raw body is currently in use.